sublime-package.json

```diff
@@ -508,7 +508,12 @@
                 "$ref": "sublime://settings/LSP#/definitions/ClientConfig"
               },
               "markdownDescription": "**DO NOT MODIFY THIS SETTING!** Use `\"clients\"` to override settings instead!",
-            }
+            },
+            "show_inlay_hints": {
+              "type": "boolean",
+              "default": true,
+              "markdownDescription": "Inlay hints are short annotations within the code, which show variable types or parameter names."
```

Comment:
This now explains what inlay hints are, which is good 👍, but it doesn’t explain what the setting does.

Reply:
Feel free to tweak the setting description.
For now I pushed, just to make it more explicit of what the setting does. [804f1c0](https://github.com/sublimelsp/LSP/pull/2018/commits/804f1c0a4c195b8bd0c7798389184c87a0a32050)

Although that setting name for me is pretty self explanatory, but that is subjective :)